kinetic/release-build.yaml

```diff
@@ -0,0 +1,62 @@
+%YAML 1.1
+# ROS buildfarm release-build file
+---
+abi_incompatibility_assumed: true
+jenkins_binary_job_priority: 81
```

Comment:
I've lowered the priority (by increasing the value) for the jobs for the older distros.